kata-deploy/README.md

```diff
@@ -10,15 +10,15 @@
     * [Remove Kata from the Kubernetes cluster](#remove-kata-from-the-kubernetes-cluster)
 * [`kata-deploy` details](#kata-deploy-details)
     * [Dockerfile](#dockerfile)
-    * [Daemonsets and RBAC](#daemonsets-and-rbac)
+    * [DaemonSets and RBAC](#DaemonSets-and-rbac)
```

Comment:
tocs/label refs don't do upper case do they? I suspect this was the victim of a global search/replace?

Reply:
correct - agreed

Reply:
Good catch - fixed!